### ISSUE TITLE
Folders are not created when local to remote.

### ISSUE THREAD

**Issue by TheBiotechScientist** [moritzgloeckl/overleaf-sync#50]
When I sync files from local to remote, files located in `capitulos` and `figuras` folders are synced and created in the main project location in *Overleaf*, but the folders `capitulos` and `figuras` are not created.

But is not the case when remote to local, folders are created.

I used the command:

```bash
~$ ols -l
```
Output:
```bash
←[KProject queried successfully.?25h
←[K←[?25h←[32m✅ ←[0m Querying project
←[KProject details queried successfully.h←[K
←[K←[?25h←[32m✅ ←[0m Querying project details
←[KProject downloaded successfully.h←[K
←[K←[?25h←[32m✅ ←[0m Downloading project
========================================

.olignore: using .olignore to filter items

Syncing files from [local] to [remote]
========================================

[NEW] Following new file(s) created on [remote]
        capitulos\antecedentes.tex
        capitulos\apendice.tex
        capitulos\conclusiones.tex
        capitulos\introduccion.tex
        capitulos\metodologia.tex
        capitulos\resultados.tex
        figuras\IPN.pdf
        figuras\IPN.svg
        figuras\pdf_file_icon.pdf
        figuras\UPIBI.pdf
        figuras\UPIBI.svg
        figuras\workinprogress.pdf
        figuras\workinprogress.svg

[UPDATE] Following file(s) updated on [remote]

[SYNC] Following file(s) are up to date

[SKIP] Following file(s) on [local] have not been synced to [remote]

✅  Synced files from [local] to [remote]
```

In *Overleaf* page, there are not folders:
![Captura de pantalla 2022-07-30 180011](https://user-images.githubusercontent.com/39642010/182002905-45c53ca5-4507-4b61-9bff-f0d712d782fc.png)

Is there an option to create/sync folders in overleaf when local to remote?

How can I solve it?

Thanks

**Comment by Lightup1** [comment]
I encountered this problem too.
And furthermore the sync from local to remote will not delete file which I locally deleted.

**Comment by moritzgloeckl** [comment]
I couldn't reproduce the folder syncing issue... Do you have any more information?

I'm currently fixing the local -> remote file deletion.

**Comment by TheBiotechScientist** [comment]
Hi @moritzgloeckl, Thanks for you replay. The issue I´m refering to is about the case when I have subfolders in the main proyect. The main proyect is in `Tesis` folder, then it has some subfolders: `capitulos`, `figuras`, for example. And when I try to sync the entire proyect, the files inside `capitulos` and `figuras` are "taken out and puted into the main folder `Tesis`" like if there are´nt subfolders inside `Tesis` folder.

In Local:
```
Tesis/
     |--- main.tex
     |--- main.aux
     |--- .... etc
     |--- capitulos/
                 |--- capitulo1.tex
                 |--- capitulo2.tex
                 |--- ... and so on.
     |--- figuras/
                 |--- figura1.pdf
                 |--- figura2.pdf
                 |--- ...etc.
```

The resulting synced from Local --> Remote (In *Overleaf*):
```
Tesis/
     |--- main.tex
     |--- main.aux
     |--- .... etc
     |--- capitulo1.tex
     |--- capitulo2.tex
     |--- ... and so on.
     |--- figura1.pdf
     |--- figura2.pdf
     |--- ...etc.
```

The command used:
```bash
C:\Users\Documents\Tesis> ols -l
```


This is my python version and the package version I have installed:
```
C:\WINDOWS\system32>python --version
Python 3.7.11

C:\WINDOWS\system32>pip show overleaf-sync
Name: overleaf-sync
Version: 1.1.6
Summary: Overleaf Two-Way Sync Tool
Home-page: https://github.com/moritzgloeckl/overleaf-sync
Author: Moritz Glöckl
Author-email: moritzgloeckl@users.noreply.github.com
License:
Location: c:\anaconda3\lib\site-packages
Requires: PyQt5, python-dateutil, PyQt5-sip, yaspin, beautifulsoup4, PyQtWebEngine-Qt5, PyQt5-Qt5, requests, PyQtWebEngine, click, socketIO-client
Required-by:

C:\WINDOWS\system32>
```

I hope you can reproduce the issue to find some solution, if not, any suggestion is well come to try myself.

**Comment by Lightup1** [comment]
This issue will not occure on linux system I think. 
For windows 11 it will upload without a folder. But in WSL ubuntu it works good.

**Comment by moritzgloeckl** [comment]
> The folder syncing issue will not occure on linux system I think. In my case: on windows 11 it will upload without a folder, but on WSL ubuntu it works good.

This is most definitely the problem. I'm developing on a Unix system as well, so I can't really test this with Windows. But I've had this problem before, thought it was fixed...

**Comment by moritzgloeckl** [comment]
Good news @TheBiotechScientist @Lightup1 - I just released version 1.2.0 I fixed both the windows syncing issue and the file deletion bug!

Please update to this version and help me test it 😉

**Comment by Lightup1** [comment]
After I updated to 1.2.0, I got this when I try to run `ols login`:
```
  Plugin uses incompatible Qt library (5.12.0) [release]
qt.core.plugin.loader: In C:/Users/dell/anaconda3/Library/plugins/platforms/qminimal.dll:
  Plugin uses incompatible Qt library (5.12.0) [release]
qt.core.plugin.loader: In C:/Users/dell/anaconda3/Library/plugins/platforms/qoffscreen.dll:
  Plugin uses incompatible Qt library (5.12.0) [release]
qt.core.plugin.loader: In C:/Users/dell/anaconda3/Library/plugins/platforms/qwebgl.dll:
  Plugin uses incompatible Qt library (5.12.0) [release]
qt.core.plugin.loader: In C:/Users/dell/anaconda3/Library/plugins/platforms/qwindows.dll:
  Plugin uses incompatible Qt library (5.12.0) [release]
qt.qpa.plugin: Could not find the Qt platform plugin "windows" in ""
This application failed to start because no Qt platform plugin could be initialized. Reinstalling the application may fix this problem.
```

**Comment by Lightup1** [comment]
I opened a new issue #59 describe `ols login` error on windows for v1.2.0
